tests/test_none.py

```diff
@@ -20,25 +20,24 @@ def test_none_as_placeholder_in_expression(
     builder = builder_class()
     module = builder.module()
 
-    block_fn_void = astx.Block()
-    block_fn_void.append(PrintExpr(astx.LiteralUTF8String("done")))
-    block_fn_void.append(astx.FunctionReturn(astx.LiteralNone()))
-
-    fn_void_proto = astx.FunctionPrototype(
-        name="fn_void", args=astx.Arguments(), return_type=astx.NoneType()
+    # tmp: int32 = 5
```

Comment:
@yuvi-mittal any reason for changing this test?
because the previous one is more complete .. it creates a function that returns None

if there is no reason for this change, please change back to the previous version